go.mod

```diff
@@ -30,7 +30,21 @@ require (
 	github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
 	github.com/aws/aws-lambda-go v1.13.3
 	github.com/aws/aws-sdk-go v1.38.60
-	github.com/aws/aws-sdk-go-v2 v0.24.0
+	github.com/aws/aws-sdk-go-v2 v1.16.2
+	github.com/aws/aws-sdk-go-v2/config v1.15.3
```

Comment:
[config/v1.15.4](https://github.com/aws/aws-sdk-go-v2/releases/tag/config%2Fv1.15.4) is the latest?